spec/mongoid/persistable/destroyable_spec.rb

```diff
@@ -297,6 +363,149 @@
         }.to raise_error(Mongoid::Errors::DocumentNotDestroyed)
       end
     end
+
+    context 'when destroying a document that was not saved' do
+      let(:unsaved_person) { Person.new(id: person.id) }
+
+      before do
+        unsaved_person.destroy!
+      end
+
+      it 'deletes the matching document from the database' do
+        lambda do
+          person.reload
+        end.should raise_error(Mongoid::Errors::DocumentNotFound, /Document\(s\) not found for class Person with id\(s\)/)
```

Comment:
For new code, we're preferring `expect { ... }.to raise_error` over the `should` form.

Reply:
Fixed. Will raise another PR to remove all shoulds from all specs everywhere.